### PR TITLE
Implementing .../users?perPage=...&page=... route

### PR DESCRIPTION
**Pull request by Gordienko-RU**
Implementation of .../users GET route, according document:
https://docs.google.com/spreadsheets/d/1Vf4ZFGRu41LPfjKUwDE8RoPu45y8T70mS_gI-ifbf_4/edit#gid=0

There is no error handler, because koa automatic will send error code 500  "Internal server error", it is enough in this case(http://koajs.com/#error-handling). 

To avoid creating multiple DB's connections has been created function, that create connection once and bind appropriate collection instance to all db handlers and return those. Then controller takes needed handlers.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const PORT = 3001;
```
Comment (dimapaloskin) [inline config.js]: Please rembeber 12 factors app. We should have ability to rewrite each variable through environment variables.

Comment (dimapaloskin) [inline config.js]: Please rembeber 12 factors app. We should have ability to rewrite each variable through environment variables.

File: controllers/users/get-users.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const getHandlers = require('../../db-handlers/users/get-handlers');
```
Comment (dimapaloskin) [inline controllers/users/get-users.js]: Why you use `controllers` name if it's basically not controllers?
`routes` is better name in your case

File: controllers/users/get-users.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+const getUsersCtrl = async ctx => {
```
Comment (dimapaloskin) [inline controllers/users/get-users.js]: Please add few empty lines in this function. It's unreadable now

File: config.js (hunk 1 above)
Comment (Gordienko-RU) [reply config.js]: Remember it, but I didn't want to store some 'public' settings into environment, only connection string with my credentials. Ok, gonna fix it.

File: controllers/users/get-users.js (hunk 2 above)
Comment (Gordienko-RU) [reply controllers/users/get-users.js]: Controller is a function that has access to request, response object and call db-handlers to implement needed operations and send response - another words, it is function that handle route. I thought it is something like this.

File: controllers/users/get-users.js (hunk 3 above)
Comment (Gordienko-RU) [reply controllers/users/get-users.js]: Ok, will make it more readable.

File: config.js (hunk 1 above)
Comment (dimapaloskin) [reply config.js]: Also, looks like you have `DB_NAME` variable here, and also you can overwrite it through connection string. Please use only one configuration point

Comment (dimapaloskin) [reply config.js]: What you mean - "public" settings?

Comment (Gordienko-RU) [reply config.js]: Public data - is a data that contains no private info, like my db url with credentials. On other hand, port that using to launch app, is not so secure. As I said, will fix it soon.

File: controllers/users/get-users.js (hunk 3 above)
Comment (Gordienko-RU) [reply controllers/users/get-users.js]: Re-wrote and even optimized that route-handler.

File: controllers/users/get-users.js (hunk 2 above)
Comment (Gordienko-RU) [reply controllers/users/get-users.js]: Changed to handlers.

File: config.js (hunk 1 above)
Comment (Gordienko-RU) [reply config.js]: Gathered those values into one .env file.

**Pull request merged**